### PR TITLE
Add scss example to postcss plugin docs

### PR DESCRIPTION
**Pull request by jeddy3**
Closes: https://github.com/stylelint/stylelint/issues/543

Sorry it took so long to get around to this one…

### CODE REVIEW

File: docs/user-guide/postcss-plugin.md [hunk 1]
```diff
@@ -62,6 +62,28 @@ gulp.task("css", function () {
...
+
+```js
+gulp.task("css", function () {
+
+  var postcss = require("gulp-postcss")
+  var reporter = require("postcss-reporter")
+  var scss = require('postcss-scss')
+  var stylelint = require("stylelint")
+
+  return gulp.src("src/**/*.css")
```
Comment (ntwb) [inline docs/user-guide/postcss-plugin.md]: Maybe `return gulp.src("src/**/*.scss")` instead?

**Comment by evilebottnawi** [comment]
@jeddy3 also all require on top in file, not inside function

File: docs/user-guide/postcss-plugin.md (hunk 1 above)
Comment (jeddy3) [inline docs/user-guide/postcss-plugin.md]: Oops. Will fix.

**Comment by jeddy3** [comment]
I, like [postcss does it](https://github.com/postcss/postcss/blob/master/gulpfile.babel.js), normally put my requires in the task as I've found it more performant on slower machines. I'd rather keep it this way as it matches the [gulp example on the postcss page](https://github.com/postcss/postcss#gulp).

**Comment by evilebottnawi** [comment]
@jeddy3 http://eslint.org/docs/rules/global-require.html

**Comment by jeddy3** [comment]
@evilebottnawi Thanks for the link.

Having the requires outside the function matches the style in [gulp's examples](https://github.com/gulpjs/gulp#sample-gulpfilejs), so my point that the previous style matches postcss's example doesn't really hold enough water to not adhere to eslint suggestion. So, I've moved the requires outside in our example :)

I'll wait for @davidtheclark feedback before merging.

**Comment by davidtheclark** [comment]
I don't give a hoot. The ESLint rule is just to enforce some people's preference: it's totally arguable. So I would be fine with the require statement inside the task; but I'm also fine with it outside. Feel free to merge. Thanks for working on this, @jeddy3.

**Comment by jeddy3** [comment]
> I'll wait for @davidtheclark feedback before merging.

I meant feedback on things like the wording of the _note_ after the example, than the require thing (as that was a minor bit of bike-shedding) :)

> Feel free to merge

Sounds like you're happy with it all though, so I'll merge now :)

**Comment by davidtheclark** [comment]
Yep, I liked the things you wrote!

**Pull request merged**